src/quo2/components/buttons/slide_button/component_spec.cljs

```diff
@@ -87,12 +87,6 @@
           small-height (:track-height constants/small-dimensions)]
       (h/has-style mock {:height small-height})))
 
-  (h/test "render with the correct customization-color"
```

Comment:
testing styles is brittle, in my opinion it will be better to add unit tests on the custom color functionality directly rather than on components using it.
I will look to add this in a separate set of work